api/Navigator.json

```diff
@@ -778,7 +778,9 @@
               "notes": "Safari 7.1.3+ uses <code>window.doNotTrack</code> rather than <code>navigator.doNotTrack</code>."
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "5.1",
+              "version_removed": "7",
+              "notes": "Safari 7.1.3+ uses <code>window.doNotTrack</code> rather than <code>navigator.doNotTrack</code>."
```

Comment:
I've tested `window.doNotTrack` on Safari 14.1 on macOS and it's not there. This and the previous note needs revising.

Reply:
I've tested with https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9402 and `window.doNotTrack` was gone by Safari 9.1, so I think we should just delete this feature-in-a-note as irrelevant.